nyc_taxis/README.md

```diff
@@ -73,6 +73,20 @@ This workload allows to overwrite the following parameters using `--workload-par
 * `error_level` (default: "non-fatal"): Available for bulk operations only to specify ignore-response-error-level.
 * `target_throughput` (default: default values for each operation): Number of requests per second, `none` for no limit.
 * `search_clients`: Number of clients that issues search requests.
+* `trip_distance_mapping` (default: { "scaling_factor": 100, "type": "scaled_float" }): The `trip_distance` field type mapping
+
+### Example of specifying parameters using `--workload-params`:
+
+Example:
+```json
+{
+  "trip_distance_mapping": { 
+    "type": "unsigned_long" 
+  } 
+}
+ ```
```

Comment:
Can you rename the section title to something like "Specifying Workload Parameters" and add a link to it from the previous section:

    This workload allows [specifying the following parameters](#specifying-workload-parameters) using the `-workload-parameters` option to OSB.

You should also indicate that if it is a simple entry, something like `--workload-params=search_clients:2` will suffice.